src/Examples.Console/Program.cs

```diff
@@ -44,7 +44,7 @@ internal class Options
 		[Option('t', "token", HelpText = "Dynatrace API authentication token with the 'metrics.ingest' permission.", Required = false)]
 		public string ApiToken { get; set; }
 
-		[Option('n', "enableDynatraceMetadataEnrichment", Default = true, HelpText = "Enables automatic label enrichment via Dynatrace metadata.", Required = false)]
+		[Option('m', "enableDynatraceMetadataEnrichment", Default = true, HelpText = "Enables automatic label enrichment via Dynatrace metadata.", Required = false)]
```

Comment:
This seems like a very long name for a flag, especially if there is no auto-completion, but i am not really sure how to shorten it. `dtMetaEnrichment` or `dtMetadataEnrichment` maybe?
Somewhat unrelated, but maybe we should also call the `exportIntervalMilliseconds` `exportIntervalMillis` instead, and add `Mins` to `duration`, as in `durationMins` (these are not consistent otherwise). WDYT?

Reply:
I was even thinking, do we need to have this enrichment in this example? What is the chance that someone is running the OneAgent locally and want to try this out? The Java one doesn't have it in the example as well: https://github.com/dynatrace-oss/opentelemetry-metric-java

When I was looking at it, it just feels like an extra thing that complicates the sample.

Reply:
I think I'd keep it. When somebody wants to use this package, they see immediately what our exporter can do and how to set it up. without having to look through the documentation. I'd assume quite a few people would actually want this feature, so id say we keep it.

Reply:
Yeah makes sense. I went with `dtMetadataEnrichment`, `exportIntervalMillis` and `durationMins. Clearer now I think :)